wormhole-connect/src/config/index.ts

```diff
@@ -19,7 +19,32 @@ const el = document.getElementById('wormhole-connect');
 if (!el)
   throw new Error('must specify an anchor element with id wormhole-connect');
 const configJson = el.getAttribute('config');
-export const config: WormholeConnectConfig = JSON.parse(configJson!) || {};
+export const config: WormholeConnectConfig = JSON.parse(configJson!) || {
+  moreTokens: {
```

Comment:
I don't think all this should be the default for connect, but rather config set by the integrator (in this case, portalbridge.com).

Reply:
removed